{{cookiecutter.project_slug}}/.neuro.toml

```diff
@@ -0,0 +1,2 @@
+[storage]
+cp-exclude = ["*.pyc", ".mypy_cache", "__pycache__", ".ipynb_checkpoints", "wandb"]
```

Comment:
Are you sure about 'wandb'? Won't this exclude affect hyperparameter tuning (as W&B probably requires some files to start new jobs during the tuning process)?

Reply:
it contains logs "*.log" and settings. Removed from the list.